sycl/doc/EnvironmentVariables.md

```diff
@@ -152,7 +152,7 @@ Note that conflicting configuration tuples in the same list will favor the last
 
 | Environment variable | Values | Description |
 | -------------------- | ------ | ----------- |
-| `SYCL_ENABLE_PCI` | Integer | When set to 1, enables obtaining the GPU PCI address when using the Level Zero backend. The default is 0. |
+| `SYCL_ENABLE_PCI` (Deprecated) | Integer | When set to 1, enables obtaining the GPU PCI address when using the Level Zero backend. The default is 0. This option is kept for compatibility reasons and is immediately deprecated. New code does not need this setting. |
```

Comment:
```suggestion
| `SYCL_ENABLE_PCI` (Deprecated) | Integer | When set to 1, enables obtaining the GPU PCI address when using the Level Zero backend. The default is 1. This option is kept for compatibility reasons and is immediately deprecated. |
```

I suggest we drop "New code does not need this setting." as it might be confusing for DPC++ users.

Reply:
thanks @bader. good suggestion. Please see latest commit.